### PR TITLE
Removed native date picker by making input type text [#94367846]

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
Hi 

Working for MIT ODL. Previously app was showing 2 date pickers on chrome browser and one date picker on reset of browsers, I have changed input type=date to type=text, now it is showing only customise date picker.

Added a validation, If user enters a date less then 100 , app with change it to **val + 2000**, means if user enters 0001-2-21 on focus out it will change it to 2001-2-21

https://github.com/jazkarta/edx-platform/issues/63
https://github.com/jazkarta/edx-platform/issues/30
https://github.com/mitocw/edx-platform/issues/21

EDX https://github.com/edx/edx-platform/pull/7946
@carsongee @pdpinch @pwilkins

screen shot 2015-05-06 at 3 47 39 pm

### CODE REVIEW

**Comment by pdpinch** [comment]
@pwilkins can you review?

File: lms/templates/ccx/schedule.html [hunk 1]
```diff
@@ -124,5 +124,18 @@
         'dateFormat': 'yy-mm-dd',
         'autoclose': true
     });
+    $('.datepair .date').change(function() {
```
Comment (carsongee) [inline lms/templates/ccx/schedule.html]: Why aren't we just using the datepickers methods for grabbing the date object? https://api.jqueryui.com/datepicker/

Comment (amir-qayyum-khan) [inline lms/templates/ccx/schedule.html]: @carsongee now using date picker api to grab date, the aim of this PR is to disallow date years like 0001,0002.... Currently date pickers allows date format like 0001-2-23 but javascrip and date picker assume it as 2001-2-23, So text field shows date  0001-2-23  but date picker dialog show it as 2001-2-23. It is ambiguous. 
in this PR I made date consistent i.e same on both date picker dialog and text field. When use adds date in text field like 0001-2-23 app convert it to 2001-2-23. A valid year.

**Comment by amir-qayyum-khan** [comment]
Fixed in https://github.com/mitocw/edx-platform/pull/29

**Pull request closed without merging**